src/main/java/com/aws/greengrass/util/Utils.java

```diff
@@ -724,21 +786,29 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
     }
 
     /**
-     * CLI params may support file URI. Detect the file URL prefix and read the file content if needed
-     * Reference: https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-file.html
-     * @param param a value or a file URL
+     * CLI params may support file URI. Detect the file URL prefix and read the file
+     * content if needed Reference:
+     * https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-parameters-file.html
+     *
+     * @param param
+     *            a value or a file URL
      * @return the same value or the file content if param is a file URI
-     * @throws URISyntaxException if provided file URI has syntax error
-     * @throws IOException on I/O Error
+     * @throws URISyntaxException
+     *             if provided file URI has syntax error
+     * @throws IOException
+     *             on I/O Error
      */
     public static String loadParamMaybeFile(String param) throws URISyntaxException, IOException {
-        return param.startsWith(FILE_URL_PREFIX) ? new String(Files.readAllBytes(Paths.get(new URI(param))),
-                StandardCharsets.UTF_8) : param;
+        return param.startsWith(FILE_URL_PREFIX)
+                ? new String(Files.readAllBytes(Paths.get(new URI(param))), StandardCharsets.UTF_8)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Server-Side Request Forgery (SSRF) vulnerability detected. User-controlled input is being used to construct HTTP requests without proper validation, which could allow attackers to make requests to internal services or arbitrary external URLs. Validate and sanitize all user inputs before using them in HTTP requests, implement allowlists for permitted destinations, and avoid direct user input in URL construction. Learn more: https://owasp.org/www-community/attacks/Server_Side_Request_Forgery.